travel_platform/travel_platform/settings.py

```diff
@@ -81,7 +81,13 @@
     }
 }
 
-
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
+        'LOCATION': '127.0.0.1:11211',
```

Comment:
Did you try to use `0.0.0.0:11211` instead?
If it works, leave it as mentioned above.